docs/contributing-to-piwik-plugins.md

```diff
@@ -0,0 +1,12 @@
+---
+category: DevelopInDepth
+---
+
+# Contributing to Matomo Plugins
+
+### Commit messages to not trigger any release.
+
+After your pull request is accepted and merged, the plugin needs to be released. Matomo tracks pushes to Matomo plugins using webhook.
```

Comment:
Maybe you mean here we keep track of all merged commits into the main branch and notify the team once a week about all changes that were made in the previous week so everyone knows what's being released and to let us know we need to release a new version for that plugin?

Reply:
updated, please check if its still confusing